core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -121,6 +120,239 @@ export class ContributorAdminStatsTable implements OnInit {
     }
   }
 
+  pluralise(num: number, word: string): string {
+    return num === 1 ? word : word + 's';
+  }
+
+  getCountDisplay(
+      count: number,
+      wordCount: number = 0,
+      editDisplay: string = ''): string {
```

Comment:
I don't know what editDisplay means ... could you perhaps make the name clearer and/or add jsdoc?

Also we try not to only start method/function names with verbs, not variable names. Is it possible to fix this?